ocaml/tests/test_bios_strings.ml

```diff
@@ -0,0 +1,97 @@
+(** This module tests the dmidecode processing
+ *)
+open Bios_strings
+
+let baseboard_two_string = "Base Board Information
```

Comment:
For long literal strings, I think `{| .. |}` is a good convention. But this will work, of course.

Reply:
Decided to stash the strings into files: they have spaces before eols, some editors might be too eager to trim the characters away.